design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
+    ): UpdatedPricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    assignBillingAddressToNegotiableQuote(
+        billingAddress: NegotiableQuoteBillingAddressInput!
+    ): AssignBillingAddressToNegotiableQuoteOutput @doc(description:"Assigns new billing address to negotiable quote")
+
+    estimateShippingCostForNegotiableQuote(
+        shippingAddress: NegotiableQuoteEstimateShippinAddressInput!
+    ): EstimateShippingCostForNegotiableQuoteOutput @doc(description:"Estimate shipping costs for a negotiable quote")
+
+    estimateShippingCostByAddressId(
+        shippingAddressId: NegotiableQuoteEstimateShippinAddressIdInput!
+    ): EstimateShippingCostByAddressIdOutput @doc(description:"Estimate shipping costs for a negotiable quote by Address ID")
+
+    estimateShippingInformationForNegotiableQuote(
+        addressInformation: NegotiableQuoteShippingInformationInput!
+    ): EstimateShippingInformationForNegotiableQuoteOutput @doc(description:"Get list of payment options and the order totals by address and shipping info")
+
+    setShippingMethodToNegotiableQuote(
```

Comment:
To be consistent with cart, please rename to `setShippingMethodsOnNegotiableQuote`. Also notice it is plural there, for potential multi-payment options. 

@mbrinton01 do we want the schema to support multiple payment methods for the same quote? It can be added in the future without modifying the schema. We have this extension point in cart schema.

Reply:
Yes, we should support for the future. Right now only a single payment method per quote is supported. It cannot be split or partially purchased.

Reply:
changed to 'setShippingMethodsOnNegotiableQuote'.

Reply:
The renamed `setShippingMethodsOnNegotiableQuote` still accepts only one method.

Reply:
This has been corrected now. Thanks

Reply:
Removed the wrapper and made it to accept multiple shipping methods.